server/common/src/main/java/org/eclipse/lsp/cobol/common/copybook/CopybookService.java

```diff
@@ -47,7 +45,6 @@ ResultWithErrors<CopybookModel> resolve(
       @NonNull CopybookName copybookName,
       @NonNull String programDocumentUri,
       @NonNull String documentUri,
-      @NonNull CopybookConfig copybookConfig,
```

Comment:
**This is a breaking change as this updates the common module**
Do we actually need a CopybookConfig ?
This is because, as a resolution strategy we always try to resolve it locally and apply for download if not found ?

Reply:
May be, but I think it's out of PR scope for now